iocore/net/quic/QUICHandshake.cc

```diff
@@ -350,6 +350,9 @@ QUICHandshake::generate_frame(QUICEncryptionLevel level, uint64_t connection_cre
 {
   QUICFrameUPtr frame = QUICFrameFactory::create_null_frame();
 
+  // CRYPTO frame is not flow-controlled
+  connection_credit = UINT64_MAX;
```

Comment:
Just comment. It looks like only StreamManager needs `connection_credit`, it'd better to make it option with default value (UINT64_MAX). But it's another cleanup.

```
QUICFrameUPtr generate_frame(QUICEncryptionLevel level, uint16_t maximum_frame_size, uint64_t connection_credit=UINT64_MAX) override;
```

Reply:
I don't think having UINT64_MAX as a default value is a good idea because 1) `QUICFrameGenerator::generate_frame` is called only from the unified block (so nobody omits it), and 2) `connection_credit` should always have a true value because the caller doesn't know which generators care about `connection_credit`.

This line doesn't make much sense because it's not accessed, but I thought overwriting this variable here would be a hint for how we should deal with `connection_credit`.

Reply:
Ah, I misunderstood. I agree with passing real connection credit value. 
( This value is passed to `QUICCryptoStream::generate_frame(QUICEncryptionLevel level, uint64_t connection_credit, uint16_t maximum_frame_size)`, though. )

I think just comment out `connection_credit` much more make sense. But it's not big deal.